SUITesslaApp/SUITesslaApp/CustomSlider/CustomSlider.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  CustomSlider.swift
+//  SUITesslaApp
+//
+//  Created by Григоренко Александр Игоревич on 01.02.2023.
+//
+
+import SwiftUI
+
+/// Кастомный слайдер c гита.
+struct CustomSlider<Value, Track, Fill, Thumb>: View
```

Comment:
```suggestion
struct CustomSlider<Value, Track, Fill, Thumb>: View
```

переименовала бы с кастового на то, какой вид у элемента или что делает и описание тоже